firecrown/likelihood/gauss_family/statistic/README.md

```diff
@@ -0,0 +1,64 @@
+# Documentation for the SACC format needed for number_counts_stats.py
+
+The Number Counts statistics object needs to be initialized with a SACC file in the right format. An example of how to create a file in such format can be found in `firecrown/examples/number_counts/Generating Cluster Data.ipynb`. Here we discuss how the file must be created.
+
+The user must provide a SACC file that only has one tracer, which shall be called `cluster_counts_true_mass`. So far, firecrown only has the implementation for real mass and real redshift. More options will be provided in the future. The tracer must be added as being the type `misc` and should be created with a directory to be added under `metadata`.
```

Comment:
Does it must have only one or *at least* one?

Reply:
Yeah, you're right.